test/contracts/LSP.spec.ts

```diff
@@ -2,28 +2,33 @@
  * @jest-environment node
  */
 
-import { ethers, getNamedAccounts } from "hardhat"
+import { config, ethers, getNamedAccounts } from "hardhat"
 import { waffleJest } from "@ethereum-waffle/jest"
 import { Address } from "hardhat-deploy/dist/types"
 import { BigNumber, Contract } from "ethers"
 import WETHAbi from "~/abis/WETH.json"
 import LSPABI from "~/abis/LSP.json"
 import LSPCreatorABI from "~/abis/LSPCreator.json"
 import ERC20ABI from "~/abis/ERC20.json"
+import OptimisticOracle from "~/abis/OptomisticOracle.json"
 import LinearLongShortPairABI from "~/abis/LinearLongShortPairFinancialProductLibrary.json"
 
 jest.setTimeout(40000)
 expect.extend(waffleJest)
 
 describe("LSP", function () {
-  const addresses: Record<string, Address> = {
+  const addresses = {
     WETH: "0xd0a1e359811322d97991e03f863a0c30c2cf029c",
     LSPCreator: "0x4C68829DBD07FEbB250B90f5624d4a5C30BBeC2c",
```

Comment:
Maybe we should use the `addresses.json` file here. I recently had the problem that the json file contained the outdated LSPCreator address leading to the issue that the redeeming worked in tests but not in the app, which was quite a confusing situation.